### DISCUSSION TITLE
Add visual indicator for DirectQuery

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1107] [Ideas]
Suggested UI enhancement. 

Have some indicator for the tables that come from a DQ source vs import, and potentially even a numerical or alphabetical indicator of the model. In the image below Employees is from a different AAS source, but other than looking at the details and compariing GUIDs no way I can see to know.

Maybe another column that has the dataset name in this view?

But even a table icon with a little link or lighting bolt to quickly distinguish form import tables.

<img width="255" alt="image" src="https://user-images.githubusercontent.com/45298358/186949888-4552c1f2-357a-4177-8b50-57a02e701b47.png">

**Comment by otykier** [comment]
Good point. DQ models and especially DQ over AS models are rarely opened in Tabular Editor, so there’s probably a bunch of stuff we can do to improve the experience overall.

**Comment by edhans** [comment]
A _pure_ DQ model I would agree, but a composite model with a mix will likely become increasingly common. I hit this today trying to debug an issue and was having trouble keeping the table origins straight.

Thanks!

**Comment by mlonsk** [comment]
This feature has been in Tabular Editor 3 for a bit:

Example with Import, but also works for DQ
<img width="827" height="104" alt="image" src="https://github.com/user-attachments/assets/a9708059-d9d7-4345-ace2-b38a06699056" />

Appologies for not updating this request earlier.
